src/components/board/TagsTabSidebar.vue

```diff
@@ -60,7 +66,7 @@
 
 import { mapGetters } from 'vuex'
 import Color from '../../mixins/color'
-import ColorPicker from '../ColorPicker'
+import { ColorPicker } from '@nextcloud/vue/dist/Components/ColorPicker'
```

Comment:
```suggestion
import { ColorPicker } from '@nextcloud/vue'
```

Since we use a lot of the components, please use the global import. That way we save quite some bytes in the bundle size.

Reply:
Ah nevermind this was from the old color picker branch ;)